librz/type/base.c

```diff
@@ -233,6 +235,16 @@ RZ_API void rz_type_base_type_free(RzBaseType *type) {
 	RZ_FREE(type);
 }
 
+/**
+ * \brief Frees the RzBaseTypeWithMetadata instance and all of its members
+ *
+ * \param type RzBaseType pointer
+ */
+RZ_API void rz_type_base_type_free(RzBaseTypeWithMetadata *type) {
+	rz_type_baes_type_free(type->base_type);
+	RZ_FREE(type->cu_name);
```

Comment:
a) Rename to `*_fini()` and add `type->base_type = NULL`
or 
b) add `free(type)` and replace `RZ_FREE(type->cu_name)` on `free(type->cu_name)`